build/common.sh

```diff
@@ -97,35 +97,27 @@ kube::build::get_docker_wrapped_binaries() {
           kube-controller-manager,busybox
           kube-scheduler,busybox
           kube-proxy,gcr.io/google_containers/debian-iptables-amd64:v3
-          federation-apiserver,busybox
-          federation-controller-manager,busybox
```

Comment:
However it must be:

1. make a code change.
2. build.
3. build/push-federation-images.sh
4. federation-up.sh